elastic/kibana
# [Uptime] monitor management - prevent showing delete toasts multiple times

Resolves https://github.com/elastic/kibana/issues/146469

## Summary

Fixes a bug where multiple popovers were displayed after a monitor was deleted from Uptime.

This fix ensures only 1 popover is displayed, and only one monitor is deleted

### Testing
1. Create a few monitors
2. Delete a monitor
3. Ensure only that monitor is deleted and only one success popover appears when deleting
